plugin/build.gradle.kts

```diff
@@ -33,15 +33,16 @@ val plugin: Configuration by configurations.creating
 configurations.getByName("compileOnly").extendsFrom(plugin)
 
 dependencies {
-    plugin("org.ow2.asm:asm:9.0")
+    plugin("org.ow2.asm:asm:9.1")
 
     testImplementation(gradleTestKit())
     testImplementation(localGroovy())
-    testImplementation("org.spockframework:spock-core:1.3-groovy-2.5")
+    testImplementation("org.spockframework:spock-core:2.0-groovy-3.0")
+    testImplementation("org.spockframework:spock-junit4:2.0-groovy-3.0")
     testImplementation("net.sourceforge.nekohtml:nekohtml:1.9.22")
-    testImplementation("org.ow2.asm:asm:8.0.1")
```

Comment:
why was the test version different from the plugin version (line 36)?

Reply:
I strongly suspect that happened by accident. I think we extract a local variable to remove the duplication and prevent it from happening again.

Reply:
Done